include/capstone/capstone.h

```diff
@@ -278,6 +278,8 @@ typedef struct cs_opt_skipdata {
 #include "evm.h"
 
 // NOTE: All information in cs_detail is only available when CS_OPT_DETAIL = CS_OPT_ON
+// Initialized as memset(., 0, offsetof(cs_detail, ARCH)+sizeof(cs_ARCH)) by ARCH_getInstruction in arch/ARCH/ARCHDisassembler.c
```

Comment:
these lines are too long, please break them into shorter lines (80 chars limit is good)